### PR TITLE
Convert a.txt to ios/android app

### PR DESCRIPTION
**Pull request by esauter5**
Hey guys,

I got tired of having to pull up github to read `a.txt`. I decided to build an ios and android app for it, so that I could instantly pull it up on me phone whenever I want. There's nothing like a nice native experience.

As this will make the top contributor, we should probably make a note of that in the README

### CODE REVIEW

File: index.android.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+class deleteme extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text>Text</Text>
```
Comment (markolson) [inline index.android.js:18]: this should probably come from a remote server so it can be updated

File: index.android.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    backgroundColor: '#F5FCFF',
+  },
+  welcome: {
+    fontSize: 20,
+    textAlign: 'center',
+    margin: 10,
+  },
+  instructions: {
+    textAlign: 'center',
+    color: '#333333',
```
Comment (markolson) [inline index.android.js:40]: Ugh. Go #444444 or go home.

**Comment by markolson** [comment]
LGTM!

**Comment by hoxworth** [comment]
tests?

**Comment by esauter5** [comment]
Added tests. As you can say they pass and are bulletproof.

screen shot 2016-03-28 at 6 19 47 pm

**Comment by hoxworth** [comment]
I can't really argue with this.

**Pull request merged**